### PR TITLE
Adapt test ODS-175 to RHODS-1.5.0

### PR DESCRIPTION
**Pull request by jgarciao**
Signed-off-by: Jorge Garcia Oncins <jgarciao@redhat.com>

### CODE REVIEW

**Comment by jgarciao** [comment]
Tested successfully here (2 tests from other tests suites failed for unrelated reasons):
https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhods-ci-pr-test/320/console

```
==============================================================================
Tests.Monitor And Manage.Metrics.Billing Metrics                              
==============================================================================
Verify OpenShift Monitoring results are correct when running undef... | PASS |
------------------------------------------------------------------------------
Test Billing Metric (notebook cpu usage) on OpenShift Monitoring      | PASS |
------------------------------------------------------------------------------
Tests.Monitor And Manage.Metrics.Billing Metrics                      | PASS |
2 tests, 2 passed, 0 failed
```

**Comment by bdattoma** [comment]
lgtm

**Review by bdattoma — APPROVED** [review]

File: tests/Tests/200__monitor_and_manage/200__metrics/201__billing_metrics.robot [hunk 1]
```diff
@@ -35,7 +38,12 @@ End Web Billing Metrics Test
   SeleniumLibrary.Close All Browsers
 
 Skip Test If Previous CPU Usage Is Not Zero
```
Comment (bdattoma) [inline tests/Tests/200__monitor_and_manage/200__metrics/201__billing_metrics.robot:40]: is this keyword needed? it seems to be not used

Comment (jgarciao) [reply tests/Tests/200__monitor_and_manage/200__metrics/201__billing_metrics.robot:39]: Removed!

**Review by lugi0 — APPROVED** [review]
lgtm

**Review by bdattoma — APPROVED** [review]

**Pull request merged**